robot/util/stresstest/README.md

```diff
@@ -1,8 +1,50 @@
 # Odroid Stress Test
 
-## Setup (optional)
+## Running the stress test (Python)
 
-This setup is optional, if you wish to test the functions on your system console at any point in time after setting up.
+If on x86 architecture, run the script with: `./odroid_stresser.py 10` for 10 seconds of runtime. If no runtime is specified, it will default to 60 seconds.
+If on arm architecture, run the script with `sudo ./odroid_stresser.py`
+
+You may stop the process at anytime with `Ctrl + c`.
+
+There will be logfiles created in the same folder as the script for the reports printed to screen, each time the script is run.
+
+Here is a sample output from running the script, on x86 architecture:
+
+```
+robotics@Tesla:~/Documents/Programming/robotics-prototype/robot/util/stresstest$ ./odroid_stresser.py 
+Checking for dependencies...
+
+stress: info: [2639] dispatching hogs: 4 cpu, 0 io, 0 vm, 0 hdd
+CPU temp: 41.000 C
+CPU freq: 2507.966 MHz
+CPU temp: 49.000 C
+CPU freq: 3192.717 MHz
+CPU temp: 50.000 C
+CPU freq: 3192.718 MHz
+CPU temp: 51.000 C
+CPU freq: 3192.718 MHz
+CPU temp: 52.000 C
+CPU freq: 3192.717 MHz
+^C
+Terminating stress test early
+
+Temperature stats:
+Min CPU temp: 41.000 C
+Max CPU temp: 52.000 C
+Avg CPU temp: 48.6 C
+
+Clock speed stats:
+Min CPU freq: 2451.913 MHz
+Max CPU freq: 3192.718 MHz
+Avg CPU freq: 3055.7672 MHz
+```
+
+## Bash Setup (optional)
```

Comment:
I'm wondering at this point, if the test is slightly outdated (and will likely be attributed no attention moving forward if additional things change), is there a point to keep it? I feel it would almost be safer to discard in case some functionality is likely to miss and considering we're using Python almost in its entirety for this project, we might as well stay true to it and keep the focus? Again, a minute detail..

Reply:
fair point
My logic for leaving it behind was because it might be convenient to use the bash functions from the console for quick testing of them, if you set up to source the bash functions in your `.bashrc` or just source the `bash_cmds.sh` if you just want to temporarily have the commands at your disposal